src/Dmux/workflow/ngs_qaqc/fastq.smk

```diff
@@ -98,7 +100,6 @@ rule kraken_annotation:
         kraken_log          = config['out_to'] + "/{project}/" + config['run_ids'] + "/{sid}/kraken/{sid}.log",
     params:
         kraken_db           = "/data/OpenOmics/references/Dmux/kraken2/k2_pluspfp_20230605"
```

Comment:
We need a method to dynamically resolve the reference files.

Reply:
Also addressed this in the next PR. I just kind of saved all the server resolution methods until I moved onto bigsky.